### PR TITLE
Reset style

### PR DESCRIPTION
**Pull request by yaowork**

### CODE REVIEW

**Comment by itswisdomagain** [comment]
Before:
screen shot 2561-11-30 at 12 42 17 am

After:
screen shot 2561-11-30 at 12 40 40 am

Does this represent the changes made @codemaestro64? Please share more screenshots if there's more...

**Comment by yaowork** [comment]
@itswisdomagain it does. Changes made as per @raedah 's request.

**Comment by raedah** [comment]
remove icons

**Comment by itswisdomagain** [comment]
When done with the requested changes, edit the title to remove [WIP] so I or anyone else available can take it up for review

**Comment by itswisdomagain** [comment]
@codemaestro64, it'd help to upload screenshots after making changes so we can have a visual rep of what was changed. Here's the latest changes I've noticed:

screen shot 2561-12-07 at 5 55 57 am

screen shot 2561-12-07 at 5 57 04 am

File: run.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```
Comment (itswisdomagain) [inline run.sh]: I suppose this is for your personal use, add this file to gitignore and remove it from git history

**Review by itswisdomagain — CHANGES_REQUESTED** [review]
All looks good, just remove the `run.sh` file from git and move the `public` folder to inside the `web` folder (related items should stay together, yeah?)

then update `web.server.go` line 34 to use the new path for the public directory like so:
`filesDir := filepath.Join(workDir, "web/public")`

**Review by itswisdomagain — APPROVED** [review]

**Pull request merged**